radicle-cli/examples/workflow/4-patching.md

```diff
@@ -0,0 +1,117 @@
+scenario.  It turns out instructions containing the power requirements were
+missing from the project.
+
+```
+$ git checkout -b flux-capacitor-power
+$ touch REQUIREMENTS
+```
+
+Here the instructions are added to the project's README for 1.21 gigawatts and
+commit the changes to git.
+
+```
+$ git add REQUIREMENTS
+$ git commit -v -m "Define power requirements"
+[flux-capacitor-power 3e674d1] Define power requirements
+ 1 file changed, 0 insertions(+), 0 deletions(-)
+ create mode 100644 REQUIREMENTS
+```
+
+Once the code is ready, we open (or create) a patch with our changes for the project.
+
+```
+$ rad patch open --message "Define power requirements" --message "See details."
+✓ Pushing HEAD to storage...
+✓ Analyzing remotes...
+
+master <- z6Mkt67…v4N1tRk/flux-capacitor-power (3e674d1)
+
+1 commit(s) ahead, 0 commit(s) behind
+
+3e674d1 Define power requirements
+
+✓ Patch a07ef7743a32a2e902672ea3526d1db6ee08108a created 🌱
+
+To publish your patch to the network, run:
+    rad push
+
+```
+
+It will now be listed as one of the project's open patches.
+
+```
+$ rad patch
+
+❲YOU PROPOSED❳
+
+Define power requirements a07ef7743a3 R0 3e674d1 (flux-capacitor-power) ahead 1, behind 0
+└─ * opened by did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (you) [..]
+└─ * patch id a07ef7743a32a2e902672ea3526d1db6ee08108a
+
+❲OTHERS PROPOSED❳
+
+Nothing to show.
+
+$ rad patch show a07ef7743a32a2e902672ea3526d1db6ee08108a
+
+Define power requirements
+
+See details.
+
+commit 3e674d1a1df90807e934f9ae5da2591dd6848a33
+Author: radicle <radicle@localhost>
+Date:   Thu Dec 15 17:28:04 2022 +0000
+
+    Define power requirements
+
+diff --git a/REQUIREMENTS b/REQUIREMENTS
+new file mode 100644
+index 0000000..e69de29
+
+```
+
+Wait, lets add a README too! Just for fun.
+
+```
+$ touch README.md
+$ git add README.md
+$ git commit --message "Add README, just for the fun"
+[flux-capacitor-power 27857ec] Add README, just for the fun
+ 1 file changed, 0 insertions(+), 0 deletions(-)
+ create mode 100644 README.md
+$ rad patch update --message "Add README, just for the fun" a07ef7743a32a2e902672ea3526d1db6ee08108a
+
+🌱 Updating patch for heartwood
+
+✓ Pushing HEAD to storage...
+✓ Analyzing remotes...
+
+a07ef7743a3 R0 (3e674d1) -> R1 (27857ec)
+1 commit(s) ahead, 0 commit(s) behind
+
+
+✓ Patch a07ef7743a32a2e902672ea3526d1db6ee08108a updated 🌱
+
+```
+
+And lets leave a quick comment for our team:
+
+```
+$ rad comment a07ef7743a32a2e902672ea3526d1db6ee08108a --message 'I cannot wait to get back to the 90s!'
+73b006df6f494e6fb9f9b9e8ad152091fc25db69
+$ rad comment a07ef7743a32a2e902672ea3526d1db6ee08108a --message 'I cannot wait to get back to the 90s!' --reply-to 73b006df6f494e6fb9f9b9e8ad152091fc25db69
+15f511ffafdc37a97703a381c9f974b52c9e8a80
+```
+
+Now, let's checkout the patch that we just created:
+
+```
+$ rad patch checkout a07ef7743a32a2e902672ea3526d1db6ee08108a
+✓ Performing patch checkout...
+✓ Switched to branch patch/a07ef7743a3
+```
```

Comment:
I'd leave this out, since it's in `rad-patch.md` and we should already be on the branch of the patch we created.

Reply:
:point_up: